core/modules/views/js/ajax.js

```diff
@@ -25,7 +25,7 @@
   };
 
   Backdrop.ajax.prototype.commands.viewsShowButtons = function (ajax, response, status) {
-    $('div.views-edit-view div.form-actions').removeClass('js-hide');
+    $('div.views-edit-view div.form-actions input.button-primary').removeClass('js-hide');
```

Comment:
I find it a bit risky to rely on a css class added by a third-party function (in this case `form_process_actions()`) to unhide the Save button.  This class is used in a lot of other context, and it's possible (thought unlikely) that this class may change at some point. 

It would probably be less risky to modify the form builder `views_ui_edit_form()` and add a custom css class to the Save submit button. Either that, or to rely on the input element ID.

Reply:
For example:

```php
$('div.views-edit-view div.form-actions input#edit-actions-save').removeClass('js-hide');
```

Reply:
I'm good with this change. It works fine in my testing. We'll see what others think.